lms/templates/courseware/courseware.html

```diff
@@ -29,6 +29,10 @@
 <%block name="headextra">
 <%static:css group='style-course-vendor'/>
 <%static:css group='style-course'/>
+## Utility: Notes
```

Comment:
That's a solid idea, but that .scss file relies on mixins/extends and patterns further up the LMS Sass compilation/cascade. So taking it out, may cause compilation and display issues. I'd say let's leave it in for now.